packages/@coorpacademy-components/src/molecule/discipline-associated-skills/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import {isEmpty, uniqueId} from 'lodash/fp';
+import Icon from '../../atom/icon';
+import ToolTip from '../../atom/tooltip';
+import Provider from '../../atom/provider';
+import CatalogSection from '../../atom/catalog-section';
+import style from './style.css';
+
+const DisciplineAssociatedSkills = (props, context) => {
+  const {translate} = context;
+
+  const {skills = [], onSkillClick} = props;
+
+  const AnchorElement = useCallback(
+    skill => (
+      <div className={style.chipFocusedContent}>
```

Comment:
Would it be simple to adapt Chip component to use it?
(seems just introducing a `position=left` kind of props would do the trick)
If so, might be cool to do so